devito/function.py

```diff
@@ -659,6 +665,7 @@ def __init__(self, *args, **kwargs):
 
             self.time_dim = kwargs.get('time_dim')
             self.time_order = kwargs.get('time_order', 1)
+            self.save = kwargs.get('save', None) is not None
```

Comment:
what's going on here?

Reply:
Flag to compare kwargs to default without forcing same time size.